src/main/java/persistence/sql/ddl/generator/H2InsertDMLGenerator.java

```diff
@@ -0,0 +1,42 @@
+package persistence.sql.ddl.generator;
+
+import persistence.sql.ddl.EntityFields;
+
+import java.lang.reflect.Field;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class H2InsertDMLGenerator implements InsertDMLGenerator{
+    @Override
+    public String generate(Object entity) {
+        EntityFields entityFields = EntityFields.from(entity.getClass());
+
+        List<String> fieldNames = entityFields.getFieldNames();
+        String columns = columnsClause(fieldNames);
+        String values = valueClause(entityFields, fieldNames, entity);
+
+        return "INSERT INTO %s (%s) values (%s);".formatted(entityFields.tableName(), columns, values);
+    }
+
+    private String columnsClause(List<String> fieldNames) {
+         return String.join(",", fieldNames);
+    }
+
+    private String valueClause(EntityFields entityFields, List<String> fieldNames, Object object) {
+        return fieldNames.stream().map(fieldName -> "'%s'".formatted(getValue(entityFields, fieldName, object)))
+                .collect(Collectors.joining(","));
+    }
+
+    private Object getValue(EntityFields entityFields, String fieldName, Object object) {
+        try {
+            Field field = entityFields.getFieldByName(fieldName);
+
+            field.setAccessible(true);
+
+            return field.get(object);
+        } catch (IllegalAccessException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
```suggestion
        Field field = entityFields.getFieldByName(fieldName);
        try {
            field.setAccessible(true);

            return field.get(object);
        } catch (IllegalAccessException e) {
            throw new RuntimeException(e);
        } finally {
            field.setAccessible(false);
        }
```
true 로 변환하여 사용 후 안전하게 이를 닫아주어도 좋을 것 같아요 😄